src/includes/sourcemaps/upload/primer/javascript.remix.mdx

```diff
@@ -0,0 +1,11 @@
+On release, call `yarn sentry-upload-sourcemaps` to upload source maps and create a release. 
+
+<Alert level="warning">
+  
+`sentry-upload-sourcemaps` requires either [`env` variables](https://docs.sentry.io/product/cli/configuration/#configuration-values) or a valid `.sentryclirc` file to pick up your project ID, organization ID and token. You can create `.sentryclirc` file with necessary configuration, as documented on this [page](https://docs.sentry.io/product/cli/configuration/).
```

Comment:
I'm not sure why this note isn't rendering properly in the preview, but I've simplified your links in the meantime. We can check how it's rendering after you've committed this change.

```suggestion
`sentry-upload-sourcemaps` requires either [`env` variables](/product/cli/configuration/#configuration-values) or a valid `.sentryclirc` file to pick up your project ID, organization ID, and token. You can create the `.sentryclirc` file with necessary configuration, as documented on this [page](/product/cli/configuration/).
```